src/pages/CreateAccount.js

```diff
@@ -44,18 +62,24 @@ export default function CreateAccount({ api, handleSuccessfulSignIn }) {
     }
 
     let userInfo = {
+      ...user,
       name: name,
       email: email,
+      learned_language: learned_language,
+      learned_cefr_level: learned_cefr_level,
+      native_language: native_language,
     };
 
-    api.addBasicUser(
```

Comment:
Will we still need the addBasicUser endpoint? 

The whole idea was that it would allow a user to be created without a Language setting, but is this something we would ever want to allow? If we decide not to have it, it does make things easier to test on the backend.

Reply:
Then I think we can delete addBasicUser, don't you agree @mircealungu ?